e2e-tests/insurance/cypress/e2e/journeys/eligibility/party-to-consortium/party-to-consortium.spec.js

```diff
@@ -0,0 +1,128 @@
+  () => {
+    const url = `${baseUrl}${PARTY_TO_CONSORTIUM}`;
+
+    before(() => {
+      cy.navigateToCheckIfEligibleUrl();
+      cy.completeCheckIfEligibleForm();
+      cy.completeExporterLocationForm();
+      cy.completeCompaniesHouseNumberForm();
+      cy.completeAndSubmitCompaniesHouseSearchForm({});
+      cy.completeEligibilityCompanyDetailsForm();
+      completeAndSubmitBuyerCountryForm({});
+      cy.completeAndSubmitTotalValueInsuredForm({});
+      cy.completeCoverPeriodForm({});
+      cy.completeUkGoodsAndServicesForm();
+      cy.completeEndBuyerForm();
+
+      cy.assertUrl(url);
+    });
+
+    beforeEach(() => {
+      cy.saveSession();
+    });
+
+    it('renders core page elements', () => {
+      cy.corePageChecks({
+        pageTitle: CONTENT_STRINGS.PAGE_TITLE,
+        currentHref: PARTY_TO_CONSORTIUM,
+        backLink: END_BUYER,
+        assertAuthenticatedHeader: false,
+      });
+    });
+
+    describe('page tests', () => {
+      beforeEach(() => {
+        cy.navigateToUrl(url);
+      });
+
+      it('renders `yes` radio button', () => {
+        yesRadio().input().should('exist');
+
+        cy.checkText(yesRadio().label(), FIELD_VALUES.YES);
+
+        cy.checkRadioInputYesAriaLabel(CONTENT_STRINGS.PAGE_TITLE);
+      });
+
+      it('renders `no` radio button', () => {
+        cy.checkText(noRadio().label(), FIELD_VALUES.NO);
+
+        cy.checkRadioInputNoAriaLabel(CONTENT_STRINGS.PAGE_TITLE);
+      });
+    });
+
+    describe('when submitting an empty form', () => {
+      beforeEach(() => {
+        cy.navigateToUrl(url);
+      });
+
+      it('should render validation errors', () => {
+        const expectedErrorsCount = 1;
+
+        cy.submitAndAssertRadioErrors({
+          field: yesRadio(FIELD_ID),
+          expectedErrorsCount,
+          expectedErrorMessage: ERROR_MESSAGES.INSURANCE.ELIGIBILITY[FIELD_ID].IS_EMPTY,
+        });
+      });
+    });
+
+    describe('when submitting the answer as `No`', () => {
+      beforeEach(() => {
+        cy.navigateToUrl(url);
+
+        cy.clickNoRadioInput();
+        cy.clickSubmitButton();
+      });
+
+      it(`should redirect to ${MEMBER_OF_A_GROUP}`, () => {
+        const expectedUrl = `${baseUrl}${MEMBER_OF_A_GROUP}`;
+
+        cy.assertUrl(expectedUrl);
+      });
+
+      describe('when going back to the page', () => {
+        it('should have the originally submitted answer selected', () => {
+          cy.clickBackLink();
+
+          cy.assertNoRadioOptionIsChecked();
+        });
+      });
+    });
+
+    describe('when submitting the answer as `yes`', () => {
+      beforeEach(() => {
+        cy.navigateToUrl(url);
+
+        cy.clickYesRadioInput();
+        cy.clickSubmitButton();
+      });
+
+      it(`should redirect to ${LONG_TERM_COVER}`, () => {
```

Comment:
Similar to above - although it looks like `LONG_TERM_COVER` is missing in the Figma flow now